compiler/src/iree/compiler/Codegen/Dialect/UKernelOps.cpp

```diff
@@ -53,7 +53,11 @@ static FailureOr<func::CallOp> createFunctionCall(RewriterBase &rewriter,
     // TODO(#12327): Based on description in the issue, add an attribute
     // `vm.import.module` and set it to `vmvx`. This only works on `vmvx`
     // backend (obviously), but is enough to unblock while the proper fix lands.
+    // For now there are a bunch of attributes set on the function, but this
+    // should be made more controllable based on the backend.
     fnDecl->setAttr("vm.import.module", rewriter.getStringAttr("vmvx"));
+    fnDecl->setAttr("llvm.bareptr", rewriter.getBoolAttr(true));
+    fnDecl->setAttr("hal.import.static", rewriter.getBoolAttr(true));
```

Comment:
we'll need to figure out a way to make dynamic cases work too - that'll be a per ukernel setting (a single executable may have some static and some dynamic ukernels) - probably the same place where we decide which object files to link in the static cases